src/org/omegat/core/plugins/PluginDowloaderGithubPackages.java

```diff
@@ -0,0 +1,56 @@
+/**************************************************************************
+ OmegaT - Computer Assisted Translation (CAT) tool
+          with fuzzy matching, translation memory, keyword search,
+          glossaries, and translation leveraging into updated projects.
+
+ Copyright (C) 2021 Hiroshi Miura
+               Home page: http://www.omegat.org/
+               Support center: https://omegat.org/support
+
+ This file is part of OmegaT.
+
+ OmegaT is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ OmegaT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **************************************************************************/
+
+package org.omegat.core.plugins;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+import org.omegat.core.threads.PluginDownloadThread;
+
+public class PluginDowloaderGithubPackages {
+    private static final String GITHUB_USER = "miurahr";  /* fixme */
+    private static final String GITHUB_TOKEN = "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";  /* fixme */
```

Comment:
The class PluginDownloderGithubPackages download jar file from Github packages repository.
The repository requires github user and its token, github gives warning source has a token.

It is not good habit to have a secret in source.

We need to seek another place to put plugin jars.